src/data/battler-tags.ts

```diff
@@ -2013,7 +2013,7 @@ export class TruantTag extends AbilityBattlerTag {
 
     const lastMove = pokemon.getLastXMoves()[0];
 
-    if (!lastMove) {
+    if (!lastMove || lastMove.move === MoveId.NONE) {
```

Comment:
```suggestion
    if (!lastMove?.move) {
```

Sliiiightly more elegant

Reply:
Doesn't the before version more closely follow the comment?

Reply:
Oh I see, thank you. I did try to read the official documentation before consulting chatGPT but was a little too stupid to understand them haha